infrastructure/cdn-in-a-box/traffic_ops_integration_test/Dockerfile

```diff
@@ -91,6 +95,9 @@ COPY --from=integration-builder \
 COPY --from=integration-builder \
     /go/src/github.com/apache/trafficcontrol/traffic_ops/testing/api/traffic_ops_v3_integration_test \
     /opt/integration/app/
+COPY --from=integration-builder \
```

Comment:
Can these `COPY --from=integration-builder`s be a single cache layer?

Reply:
Consolidated them to a single cache layer in 6c3507ac8c.